### PR TITLE
task/JM-7183 bump up test coverage

### PR DESCRIPTION
**Pull request by T-Sarwar**
**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[X ] No

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java [hunk 1]
```diff
@@ -267,6 +267,53 @@ void positiveTypical() throws IOException, SQLException {
...
+                    "Expect 0 files to be updated successfully");
+                assertEquals("3", result.getMetaData().get("TOTAL_FILES_UPDATED_UNSUCCESSFULLY"),
+                    "Expect 3 files to be updated unsuccessfully");
+                assertEquals("0", result.getMetaData().get("TOTAL_FILES_UPLOADED_UNSUCCESSFULLY"),
+                    "Expect 0 files to be uploaded unsuccessfully");
+
+                for (File file : files) {
+                    verify(sftpService, times(1)).upload(sftpClass, file);
+                    fileUtilsMock.verify(() -> FileUtils.deleteFile(eq(file)), times(1));
+                    verify(databaseService, times(3))
```
Comment (Ben-Edwards-cgi) [inline src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java:311]: Why do we upload three times per file here?

This should be once per file

File: src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java [hunk 2]
```diff
@@ -275,15 +285,24 @@ void updateSqlException() throws IOException, SQLException {
...
+                Set<File> files = new HashSet<>(Set.of(file1, file2, file3));
+
                 when(fileSearch.search()).thenReturn(files);
                 when(fileSearch.setFileNameRegexFilter(any())).thenReturn(fileSearch);
                 when(sftpService.upload(eq(sftpClass), any(File.class))).thenReturn(true);
 
                 SQLException sqlException = new SQLException("Forced sql exception");
                 for (File file : files) {
+                    String fileName = file.getName();
+                    when(file.getName()).thenReturn(fileName);
```
Comment (Ben-Edwards-cgi) [inline src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java:303]: This is not needed as it is already mocked

File: src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java [hunk 3]
```diff
@@ -275,15 +285,24 @@ void updateSqlException() throws IOException, SQLException {
                 FileSearch fileSearch = mock(FileSearch.class);
                 fileSearchMock.when(() -> FileSearch.directory(ftpDirectory, true)).thenReturn(fileSearch);
 
-                Set<File> files = new HashSet<>(Set.of(mock(File.class), mock(File.class), mock(File.class)));
+                File file1 = Mockito.mock(File.class);
+                Mockito.when(file1.getName()).thenReturn("sample1.txt");
```
Comment (Ben-Edwards-cgi) [inline src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java:289]: Does not need to be fully qualified can just be when(...)

File: src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java (hunk 2 above)
Comment (T-Sarwar) [reply src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java:303]: Removed

File: src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java (hunk 3 above)
Comment (T-Sarwar) [reply src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java:289]: Removed

**Review by Ben-Edwards-cgi — APPROVED** [review]

**Pull request merged**